contributors/devel/sig-release/flake-finders/episodes/000/README.md

```diff
@@ -0,0 +1,141 @@
+# Flake Finder Fridays #0 
+
+February 5th 2021 ([Recording](https://youtu.be/Hqlm2h2AEvA))
+
+## Introduction
+
+This is the first episode of Flake Finder Fridays with Dan Mangum and Rob
```

Comment:
Maybe include your GitHub usernames here?

Reply:
@justaugustus good idea :) updated :+1: could you re-add your LGTM here?